integration-tests/src/test/resources/domain-home-on-pv/create-domain-auto-custom-sit-config.py

```diff
@@ -48,17 +48,17 @@ def getEnvVar(var):
 # Configure the Administration Server
 # ===================================
 cd('/Servers/AdminServer')
-# Give incorrect listenaddress, introspector overrides with sit-config
-set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
+# Dont set listenaddress, introspector overrides automatically with sit-config
+#set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
```

Comment:
Do we have formal 'p4 tests' or similar that check (a) setting admin server & managed server listen address/nap address to a completely invalid DNS address (one that doesn't resolve)? (b) or to a DNS address that resolves to an incorrect location ('oracle.com' would probably work internally and externally as an example)?

Reply:
I don't think so. If you want to cover these use cases in java tests, I can create a JIRA for this.